src/composables/draw/draw-utils.ts

```diff
@@ -9,14 +14,36 @@ import { Feature } from 'ol'
 // TODO 3D
 // const ARROW_MODEL_URL = import.meta.env.VITE_ARROW_MODEL_URL
 
-function convertCircleToPolygon(
-  feature: Feature<Geometry>,
-  featureType: String
-) {
+function convertCircleFeatureToPolygon(feature: DrawnFeature): DrawnFeature {
   const geom = feature.getGeometry()
-  if (featureType == 'drawnCircle' && geom?.getType() == 'Circle') {
+  if (feature.featureType == 'drawnCircle' && geom?.getType() == 'Circle') {
```

Comment:
```suggestion
  if (feature.featureType === 'drawnCircle' && geom?.getType() === 'Circle') {
```

Reply:
So in which case should we have feature with feature.featureType == 'drawnCircle' and geom?.getType() == 'Polygon'? for export maybe? (just asking, maybe we should add some explanation in the js doc function above the function decleration)

Reply:
Indeed, it should not occur when the function is called, but the Circles are managed as polygon geometries in the URL and during export, so the case could occur when the function is called at the wrong time. I added a comment.